src/features/user/Profile/components/AddTargetModal.tsx

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
-import Modal from '@material-ui/core/Modal';
-import Backdrop from '@material-ui/core/Backdrop';
+import Modal from '@mui/material/Modal';
```

Comment:
Slight difference in font size and other font styles in the modal. Especially noticeable for heading.

Reply:
Fixed Here https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1467/commits/899022f498edbef1e2c102fc2fae950fb381618a

Reply:
Resolved in #1467